include/silo/query_engine/operator_result.h

```diff
@@ -0,0 +1,21 @@
+#ifndef SILO_OPERATOR_RESULT_H
+#define SILO_OPERATOR_RESULT_H
+
+#include <roaring/roaring.hh>
+
+namespace silo::query_engine {
+
+/// The return value of the Operator::evaluate method.
+/// May return either a mutable or immutable bitmap.
```

Comment:
Why don't we put that struct into `operator.h`? Then we could simply get rid of that doc comment.

Reply:
I do not know yet how complicated it will become in future refactors